src/logic/__tests__/startAccountMigration.test.js

```diff
@@ -0,0 +1,107 @@
+import {Observable} from 'rxjs';
+
+import startAccountMigration from '../startAccountMigration';
+import {
+  accountMigrationComplete,
+  accountMigrationError,
+  accountMigrationUndoPeriodExpired,
+} from '../../actions/user';
+import {migrateAccount} from '../../clients/firebase';
+import {getCurrentAccountMigration} from '../../selectors';
+import {bugsnagClient} from '../../util/bugsnag';
+
+import {
+  credentialFactory,
+  firebaseErrorFactory,
+  userFactory,
+} from '@factories/clients/firebase';
+
+import {
+  githubRepositoryFactory,
+} from '@factories/clients/github';
+
+
+jest.mock('../../actions/user.js');
+jest.mock('../../clients/firebase.js');
+jest.mock('../../selectors');
```

Comment:
Is this necessary?

Reply:
Yes.

Reply:
Even in the context of [this](https://github.com/popcodeorg/popcode/pull/1701#discussion_r291980415)?

Reply:
I wrote that comment before I read that part ;). Working on fixing that then I'll adapt.

Reply:
I don’t think we need this mock?